### DISCUSSION TITLE
Change from SuperSlicer to BambuStudio/SoftFever

### DISCUSSION THREAD

**Discussion by 3D-Frank** [OrcaSlicer/OrcaSlicer#172] [Ideas]
Hello! 
First of all I would like to thank you for this successful further development of the Bambu Slicer. Yesterday I was able to make the first positive experiences with the initial setup on my iMac/intel. Up until now I've mainly worked with the related Superslicer. 
And now to my questions/suggestions:
Is it possible to import my labour-intensive configurations from Superslicer into BambuStudio? This simplification would certainly bring many more 3D friends into your boat.
I started using the Prusa printer (I read your answer about Voron/Ratrig too late) as a template for my AnycubicVyper with Klipper and I'd still like to change the look of the print bed. Is that possible and if so where can I find suitable files? It can also be neutral or a matching Voron look.
Or is it better that I start the setup from scratch and then use a Voron as a template.
But then I would probably have to start over with my previous settings?

Thank you in advance!

**Comment by hliebscher** [comment]
is it better that you start the setup from scratch and use any of the new printer settings 

it works for me very good

**Comment by leezus-nz** [comment]
> is it better that you start the setup from scratch and use any of the new printer settings

I agree, but would also like to see an import tool.

Fortunately for me, SoftFever profiles are better than most of what I had in SuperSlicer after a couple of years anyway, so it's a good (better?) starting point.
